doc/changes/1924.misc.py

```diff
@@ -0,0 +1 @@
+Return TypeError instead of Exception for type error in sesolve argument
```

Comment:
Thank you for adding the changelog with towncrier -- I think you are officially the first person to use this (after the PR that added it).

I think the filename should be just "1924.misc" though, without the `.py`.

Reply:
I was already wondering why the `changes` folder was so empty :-)